paper/Manuscript.Rmd

```diff
@@ -379,6 +379,11 @@ If all collaborators are committed to using WORCS, they can [Fork the repository
 
 WORCS offers a workflow for open reproducible code in science. The step-by-step procedure outlined in this tutorial helps researchers make an entire research project Open and Reproducible. The accompanying R-package provides user-friendly support functions for several steps in the workflow, and an RStudio project template to get the project set up correctly.
 
+## Outlook
+
+<!-- not sure where this repro integration section belongs -->
+The `worcs`-package integrates several reproducibility tools into a convenient workflow by assuming reasonable standards that fulfil most researches needs. Currently, however, if a researcher prefers to use other tools that are conceptually similar, they have to leave WORCS behind. We currently integrate WORCS into a modular framework of reproducibility tools for R [@repro], which will serve as the backend for WORCS. This ongoing integration into a unifying framework does not compromise the ease of use for the researcher. However, it gives them more flexibility to satisfy the demands of their research projects while meeting the TOP-guidelines.
```

Comment:
Is it true that researchers have to leave worcs behind? You cannot simply instantiate a worcs project inside a repro project?

Reply:
Technically that is possible (I'd have to test but you just have to deactivate `renv`), it is more meant like you are mentally back where you were before WORCS. You have to know what components does what, how they have to be set up or learn how to use repro.

So maybe: "Currently, however, if a researcher prefers to use other tools that are conceptually similar, they have to integrate them themself into their workflow."?

Reply:
Are you sure renv doesn't work in that environment? :(

Reply:
Now that I am thinking about it, `renv` works inside repro projects. Even together with docker renv works just fine, so you could instantiate a worcs project, then load repro and use `use_make()` and `use_docker()` just fine.

Reply:
We should test that, and put it in the paper!
PS: Also multiple data files now works, as long as they are flat csv files.

Reply:
@aaronpeikert can we commit to readers of the paper that they will be able to add make and docker in the way you describe? I think that will make it more convincing.